tests/utils/decorators/test_decorators.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Import modules
+import pytest
+import numpy as np
+from numbers import Number
+
+# Import from package
+from pyswarms import cost
+
+
+@pytest.mark.parametrize(
+    "objective_func",
+    [np.sum(), np.prod()]
+)
+def test_cost_decorator(objective_func, particles):
```

Comment:
Where do you get the particles? I suggest you can generate them via `np.random.uniform`...

Reply:
Do you know a way we can output the seed number with the tests without using `pytest -s`? In this way, we don't have fixed particle arrays and if it fails we can use the printed seed to debug it.